packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -63,6 +63,12 @@ export const freshAtlassianAuth = (parent: RootDataLoader) => {
                 refreshToken: updatedRefreshToken
               }))
             await upsertAtlassianAuths(updatedSameJiraAccountAtlassianAuths)
+
+            return {
+              ...atlassianAuthToRefresh,
+              accessToken,
```

Comment:
nit: rename this to `updatedAccessToken` to make it clear it's the new one, and also matches the naming pattern with `updatedRefreshToken`